docs/_build/html/_sources/home.rst.txt

```diff
@@ -52,20 +52,78 @@ To make any changes in the code follow the below steps :
+
+* To change the designing of the card - Go to the file `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__ and search :python:`.flip-card-front` and do the desired changes.
+
+ 
+5. **Our Achievements**: This section describes the achievements by the girlscript. It is a card which is divided into 3 parts.
+
+.. image:: ./images/homepage/achievements.png
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+To make changes in the left part follow the below steps :
+
+- To make changes in the content - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`achieve-content`, and make changes in the <h3> tag.
+
+* To make changes in the path of button - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`achieve-content` , you will see <a> tag where in href you can give the path of the file.
 
-5. **Our Achievements**:
+- To make changes in the CSS of the button -  Go to the file `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__ and search :python:`.center-btn-settings` or :python:`.center-btn-settings2` and do the desired changes. 
 
-6. **Various Trends**:
+To make changes in the right part follow the below steps :
+
+- To make changes in the number count - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`ci-number`, and make changes inside the div container and similarly for other number count also.
+
+* To make changes in the heading and content - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`ci-text`, for heading <h4> tag and for content <p> tag and then do the desired changes accordingly.
+
+- To make changes in the box shadow of the container - Go to the file `style.css <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/css/style.css>`__ and search :python:`.myDiv` where you can change the box-shadow by giving appropriate size and color to it.
+
+To make any changes in the animation effect of the whole section - Go to `index.html <https://github.com/smaranjitghose/girlscript_chennai_website/blob/master/index.html>`__ and search :python:`sparkley` or :python:`sparkle-canvas`, do your desired changes whether you want or not or any style.
+
+
+
+6. **Various Trends**: This section covers various trends upon which contributions have been made.
+
+.. image:: ./images/homepage/trends.png
+  :width: 600
+  :align: center
+  :alt: Alternative text
+
+To make changes in the code follow the below steps :
```

Comment:
Rewrite as:

If you want to make changes in this section, carry out the following changes in code: 

(Repeat for other such lines throughout this page)

Reply:
okay sir